### ISSUE TITLE
OBS Window doesnt remember Windowsize on Win11

### ISSUE THREAD

**Issue by Rockatweb** [obsproject/obs-studio#7510]
### Operating System Info

Windows 11

### OBS Studio Version

Other

### OBS Studio Version (Other)

28.0.3

### OBS Studio Log URL

https://obsproject.com/logs/0pVSN7Y4vvEhGQGs

### Expected Behavior

The OBS Window should remember the size from the last session or should start maximized

### Current Behavior

The Window is around 75% width from the total screen width

### Steps to Reproduce

1. just open obs on Windows 11

### Anything else we should know?

i have locked the docks but because of the resizing on opening obs all windows are resized automatically.
The bug is by the way not because i use the windows insider build. I had the problem before when i was on the normal realease channel

**Comment by RytoEX** [comment]
Is this a duplicate of any of these:
* #2949
* #4666
* #6451

**Comment by ntoff** [comment]
I have a similar issue on windows 10. Sometimes when I open obs, the main window will be "maximized" according to the maximize icon, but the window only takes up about 75% of the width of the screen. Didn't bother reporting it because I can't figure out how to replicate it, it seems totally random for me.

The dock layout / sizes remain the same for me, I just minimize/maximize the window and it's back to normal.

**Comment by Rockatweb** [comment]
> Is this a duplicate of any of these:
> 
>     * [Dock Sizes are Reset the next time you open OBS. #2949](https://github.com/obsproject/obs-studio/issues/2949)
> 
>     * [Resizing OBS window causes dock layout to shift #4666](https://github.com/obsproject/obs-studio/issues/4666)
> 
>     * [Dock Sizes reset when OBS reopens with "Minimise to system tray when started" selected #6451](https://github.com/obsproject/obs-studio/issues/6451)

nope, it is a different problem. Like i wrote my OBS opens with a window size about 75% of the screen size.

**Comment by Jinxyface** [comment]
Windows 10, OBS Studio 28.1.2. If OBS is maximized on my 2nd display, closing and re-opening will reset it and launch on my main display un-maximized. If I put it on my 2nd display but keep it windowed, closing and re-opening will remember window size/position and open on my 2nd monitor.

I don't THINK this was hapening in v27, but it seems to happen in 28.x.x

**Comment by RytoEX** [comment]
Please provide a more detailed Steps To Reproduce. As-is, this is not actionable. Are you maximizing OBS, then closing and reopening it? Are you manually resizing OBS to be a specific size before closing and reopening it? Can you reproduce this without any third-party plugins installed? Do you have any service accounts connected (e.g., Twitch, YouTube)?

**Comment by Rockatweb** [comment]
for me the problem is fixed. i dont know if you fixed it or a windows update but it works now
